pkgs/llvm_cheriot.nix

```diff
@@ -0,0 +1,28 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+{pkgs, ...}:
+pkgs.stdenv.mkDerivation rec {
+  name = "llvm-cheriot";
+  src = pkgs.fetchFromGitHub {
+    owner = "CHERIoT-Platform";
+    repo = "llvm-project";
+    rev = "432ef65c96c9f8be8b7802a8bfb9bcf3609b2b79";
+    sha256 = "sha256-ddLnG1olyRYxvKJ0zKEbThq/ZnxP68X32+1ws76WeV4=";
+  };
+  sourceRoot = "${src.name}/llvm";
+  nativeBuildInputs = with pkgs; [cmake ninja lld python3];
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DLLVM_ENABLE_ASSERTIONS=True"
+    "-DLLVM_ENABLE_PROJECTS='clang;clang-tools-extra;lld'"
+    "-DLLVM_ENABLE_UNWIND_TABLES=NO"
```

Comment:
Where's this line coming from?

Reply:
It came from https://github.com/microsoft/cheriot-rtos/blob/ff645af8108282622976e2d7fd42c193241e5ce0/docs/GettingStarted.md?plain=1#L128